js/widgets/fixedToolbar.js

```diff
@@ -37,59 +39,54 @@
 
 return $.widget( "mobile.toolbar", $.mobile.toolbar, {
 	options: {
-		position: null,
+		position:null,
 		visibleOnPageShow: true,
 		disablePageZoom: true,
-		transition: "slide", //can be none, fade, slide (slide maps to slideup or slidedown)
+
+		// Can be none, fade, slide (slide maps to slideup or slidedown)
+		transition: "slide",
```

Comment:
@cgack - You are right.

On the API Docs Reference page we say "Show/hide-transition when a tap/click occurs". We have to change that because it sounds like it only is meant for ```tapToggle```.